packages/core/js-client/src/index.ts

```diff
@@ -141,9 +109,10 @@ const createClient = async (
 
   const client = new ClientPeer(peerConfig, relayConfig, keyPair, marine);
 
-  if (isNode) {
-    doRegisterNodeUtils(client);
-  }
+  // TODO: Support node specific utils
+  // if (isNode) {
+  //   doRegisterNodeUtils(client);
+  // }
```

Comment:
this is breaking if I am not mistaken. Why is this suddenly not supported?

Reply:
Earlier it would fail at runtime if frontend tried to call this method accidentally.
Also frontend bundlers doesn't feel good to add node.js dependencies in bundle.
Currently i commented this out because there is a chance to rethinking this approach. For example we can add `fs` polyfill to the browser and would have exactly same code in both environments.

Reply:
it's ok. maybe have a task for that. but it still seems breaking right? it will stop working if node.js user uses this while it was previously working for them

Reply:
Yes, unfortunately it will break.

Reply:
just mark the PR as breaking in this case. And maybe entirely rename it. You wanted to just update README and ended up with breaking changes, maybe some more useful commit name would be great to have